tests/testRandomOps.lua

```diff
@@ -35,10 +36,10 @@ end
 function Counter(l)
   local ret = {}
   for k,v in pairs(l) do
-    if ret[k] then
-      ret[k] = ret[k] + 1
+    if ret[k..""] ~= nil then

```

Comment:
Forces it to be a string so that it index's the table portion of the object instead of the array, which only exists on half of the objects being iterated over.